qgreenland/config/helpers/layers/continental_shelf.py

```diff
@@ -0,0 +1,96 @@
+from qgreenland.config.datasets.continental_shelf import continental_shelf as dataset
+from qgreenland.config.helpers.steps.ogr2ogr import STANDARD_OGR2OGR_ARGS
+from qgreenland.models.config.layer import Layer, LayerInput
+from qgreenland.models.config.step import CommandStep
+
+
+LAYER_PARAMS = {
+    'north_points': {
+        'title': 'North (points)',
+        'description': (
+            """Northern continental shelf of Greenland."""
```

Comment:
TODO:

- [ ] For each description, more clearly describe what the user is looking at. e.g. "Points representing the boundary of the Northern extended continental shelf of Greenland."

Twila asked about the polygons -- what does the area within the polygon mean? Maybe we should remove the polygons if the area within the polygon is unimportant. We should probably follow-up with Vinton. We shouldn't wait for a response from him before merging this, just planning ahead. Let's do a first draft on a better description before merge though.